docs/src/playground/SearchPage.tsx

```diff
@@ -0,0 +1,60 @@
+import { Grid, useTheme } from '@mui/material';
```

Comment:
Is this the intended desktop view?  I didn't see a design in the Figma for one, and this differs greatly from the older versions.

Reply:
Thanks Ed, I'd forgotten that for desktop, we have a different layout :)

Reply:
![Screen Shot 2022-11-30 at 7 29 25 AM](https://user-images.githubusercontent.com/106985561/204801281-506899c9-010e-49a7-9dc7-500987e75778.png)
![Screen Shot 2022-11-30 at 7 29 43 AM](https://user-images.githubusercontent.com/106985561/204801295-ba6bf9f2-acb2-4752-a189-eed8e0153081.png)